packages/console/src/scss/modal.module.scss

```diff
@@ -4,17 +4,17 @@
   position: fixed;
   background: rgba(0, 0, 0, 40%);
   inset: 0;
-  overflow-y: auto;
-  padding: _.unit(12) 0;
+  overflow: auto;
+  padding: _.unit(12);
 }
 
 .content {
-  display: flex;
```

Comment:
why `inline-flex`?

Reply:
if we use `display: flex; flex-direction: column;`, the `content` div width will not auto grow if the `modal layout`'s width is bigger than this `content` in the cross-axis direction. This causes a scrolling problem: we can not scroll to the left-most of the `modal layout`.
When changed to `display: inline-flex;`, the `content` div will auto fit the with of the `modal layout`, and meanwhile, we add `min-width: 100%` to make sure the `modal layout` will be in the center of the screen if the width of it is small than that of the content.